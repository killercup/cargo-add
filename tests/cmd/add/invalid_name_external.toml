bin.name = "cargo-add"
args = ["add", "lets_hope_nobody_ever_publishes_this_crate"]
status.code = 1
fs.sandbox = true
