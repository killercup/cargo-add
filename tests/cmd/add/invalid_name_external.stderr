    Updating 'https://github.com/rust-lang/crates.io-index' index
Command failed due to unhandled error: The crate `lets_hope_nobody_ever_publishes_this_crate` could not be found in registry index.

